data/xml/W19.xml

```diff
@@ -1049,7 +1049,7 @@
       <address>Gothenburg, Sweden</address>
       <month>June</month>
       <year>2019</year>
-      <venue>iwcs</venue>
+      <venue>cstfrs</venue>
```

Comment:
Can you elaborate on this @mcandito? It has IWCS in the title; is IWCS not its primary affiliation? We can have a joint affiliation if that is the case (in which case, there will be two `<venue>` tags).